.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2.1
+
+jobs:
+  test:
+    docker:
+      - image: circleci/python:latest
+    working_directory: ~/repo
+    parameters:
+      python:
+        type: string
+      django:
+        type: string
+    steps:
+      - checkout
+      - run: pip install tox
+      - run: tox -e "py<< parameters.python >>-django<< parameters.django >>"
+
+workflows:
+  default-workflow:
+    jobs:
+      - test:
+          matrix:
+            parameters:
+              python: ["36", "37", "38", "39"]
```

Comment:
haha, then maybe `type: number` would have worked?

Reply:
Probably, yeah. I figured it was more of a string than a number in reality, so I went with that.